charts/telegraf/README.md

```diff
@@ -39,6 +39,30 @@ helm uninstall telegraf
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## livenessProve, readinessProbe and health service
+
+Be sure to have at least these inputs and outputs enabled so that readinessProbe and livenessProbe can work as expected and that the health service endpoint is correctly set up:
+
+```yaml
+[...]
+
+config:
```

Comment:
Can't we enforce this?

Reply:
Indeed, we can - I thought we could not when I first implemented it ;-)

Reply:
done